Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'jquery-rails'
 gem 'turbolinks'
 gem 'jbuilder', '~> 2.0'
 gem 'sdoc', '~> 0.4.0', group: :doc
+gem "haml-rails", "~> 0.9"
```

Comment:
ja uzywam gem 'slim-rails' ale szczerze nie wiem czym różni sie zwykly slim od slim-rails :D i chyba to mało znaczące